UIFont+Montserrat.m

```diff
@@ -7,47 +7,56 @@
 #import <CoreText/CoreText.h>
 #import "UIFont+Montserrat.h"
 
+@interface KOSFontLoader : NSObject
 
-@implementation UIFont (OpenSans)
++ (void)loadFontWithName:(NSString *)fontName;
 
-void MontserratLoadFontWithName(NSString *fontName) {
-    NSURL *bundleURL = [[NSBundle mainBundle] URLForResource:@"Montserrat" withExtension:@"bundle"];
+@end
+
+@implementation KOSFontLoader
+
++(void) loadFontWithName:(NSString *)fontName {
+    NSURL *bundleURL = [[NSBundle bundleForClass:[self class]] URLForResource:@"Montserrat" withExtension:@"bundle"];
     NSBundle *bundle = [NSBundle bundleWithURL:bundleURL];
     NSURL *fontURL = [bundle URLForResource:fontName withExtension:@"ttf"];
     NSData *fontData = [NSData dataWithContentsOfURL:fontURL];
-
+    
     CGDataProviderRef provider = CGDataProviderCreateWithCFData((CFDataRef)fontData);
     CGFontRef font = CGFontCreateWithDataProvider(provider);
-
+    
     if (font) {
         CFErrorRef error = NULL;
         if (CTFontManagerRegisterGraphicsFont(font, &error) == NO) {
             CFStringRef errorDescription = CFErrorCopyDescription(error);
             @throw [NSException exceptionWithName:NSInternalInconsistencyException reason:(__bridge NSString *)errorDescription userInfo:@{ NSUnderlyingErrorKey: (__bridge NSError *)error }];
         }
-
+        
         CFRelease(font);
     }
-
+    
     CFRelease(provider);
 }
 
-+ (instancetype)montserratLoadAndReturnFont:(NSString *)fontName size:(CGFloat)fontSize onceToken:(dispatch_once_t *)onceToken fontFileName:(NSString *)fontFileName {
```

Comment:
These methods we're purposely prefixed with the font name because they are extensions on a public class which could cause conflicts (such as with other fonts also implementing the `loadAndReturnFont:size:onceToken: fontFileName:` method.

Can you restore these changes as they don't make sense.